release.nix

```diff
@@ -126,6 +126,8 @@ let
     [ "cardano-node-profiled" "components" "exes" ]
     [ "checks" "tests" "cardano-node-chairman" ]
     [ "workbench" ] [ "profiles" ]
+    [ "haskellPackages" "cardano-node-chairman" "checks" ] [ "checks" "tests" "cardano-node-chairman" ]
```

Comment:
@MarcFontaine, also slipped by.

Reply:
This is unclear.
Did I add that line by mistake ?

Reply:
This is what this PR adds to the baseline, indeed -- and it shouldn't, since this line is completely unrelated to the generator.